helmfile.d/helmfile-91.artifacts.yaml

```diff
@@ -0,0 +1,26 @@
+bases:
+  - snippets/defaults.yaml
+---
+bases:
+  - snippets/env.gotmpl
+---
+bases:
+  - snippets/derived.gotmpl
+---
+{{ readFile "snippets/templates.gotmpl" }}
+{{- $v := .Values }}
+{{- $a := $v.apps }}
+
+releases:
+  - name: otel-operator-artifacts
```

Comment:
This release is already defined in the `helmfile.d/helmfile-90.artifacts.yaml` file

Reply:
indeed